### PR TITLE
Nav bar

### PR DESCRIPTION
**Pull request by josh-hdz**

### CODE REVIEW

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -134,26 +132,42 @@ header p:first-child{
     margin-bottom: 1%;
 }
 
-/*  Main
+/*  Navegation
```
Comment (jmuzy-sps) [inline portfolio/src/main/webapp/style.css]: nit: spelling (navigation)

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -134,26 +132,42 @@ header p:first-child{
...
 ------------------------------------*/
-main {
-    background: var(--cultured);
+nav {
+    background: rgba(var(--cultured-rgb), 1.0);
+    left: 0;
+    top: 0;
+    position: fixed;
+    text-align: center;
+    width: 100%;
```
Comment (jmuzy-sps) [inline portfolio/src/main/webapp/style.css]: alphabetical order

**Review by jmuzy-sps — CHANGES_REQUESTED** [review]
Nice job, two super minor comments!

File: portfolio/src/main/webapp/style.css (hunk 1 above)
Comment (josh-hdz) [reply portfolio/src/main/webapp/style.css]: Spanglish hahaha

**Review by jmuzy-sps — APPROVED** [review]

**Pull request merged**